### DISCUSSION TITLE
Feature Request : Can we please get the option to Customize the “Home” tab section by completely hiding the “Shortcut’s” section on the mobile app on iOS. Please!!!

### DISCUSSION THREAD

**Discussion by Gta211** [community/community#146060] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.187.0 (147865513)
Device: iPhone 14 Pro Max (iOS 18.1.1)
</details>

**Comment by stevepopovich** [comment]
Hi @Gta211 thanks for the request. I have passed this onto the team for prioritization!
